components/org.wso2.carbon.stream.processor.common/src/main/java/org/wso2/carbon/stream/processor/common/utils/config/RootConfiguration.java

```diff
@@ -26,13 +26,17 @@
 /**
  * {@code RootConfiguration} is a bean class for root level configuration.
  */
-@Configuration(namespace = "siddhi", description = "Siddhi extension parent level configuration")
+@Configuration(namespace = "siddhi", description = "Siddhi extension and store parent level configuration")
 public class RootConfiguration {
 
     @Element(description = "Extension list", required = true)
     public List<Extension> extensions;
 
+    @Element(description = "Store list", required = false)
+    public List<Store> stores;
```

Comment:
Any reason to use public access modifier?

Reply:
Fixed in 279c00f